### PR TITLE
first spin at TLS certificate support - works for me

### PR DESCRIPTION
**Pull request by gdhgdhgdh**
I learned a little Go and this PR is working for me against VerneMQ in TLS mode.

### CODE REVIEW

**Comment by arnisoph** [comment]
@gdhgdhgdh thx! we'll check it!

File: main.go [hunk 1]
```diff
@@ -11,6 +11,8 @@ import (
 	"net/http"
 	"os"
 	"time"
+    "crypto/tls"
```
Comment (arnisoph) [inline main.go]: could you fix the whitespacing here? :)

Comment (arnisoph) [reply main.go]: tip: ``make fmt``

**Comment by arnisoph** [comment]
@frittentheke could you test this PR please?
@hikhvar / @invadersmustdie could you assist on reviewing the code please?

**Review by hikhvar — APPROVED** [review]
Looks good for me.

**Comment by invadersmustdie** [comment]
lg2m

**Pull request merged**